src/main/java/org/sonarlint/intellij/actions/MarkAsResolvedAction.kt

```diff
@@ -60,13 +61,18 @@ class MarkAsResolvedAction(
     private var issue: Issue? = null
 )  :
     AbstractSonarAction(
-        "Mark as Resolved", "Mark as Resolved", null
+        "Mark Issue as...", "Change the issue resolution status", null
     ), IntentionAction, PriorityAction, Iconable {
     companion object {
         private const val errorTitle = "<b>SonarLint - Unable to mark the issue as resolved</b>"
         private const val content = "The issue was successfully marked as resolved"
```

Comment:
Nitpick -> We could put the actual status in the message so the user has more feedback on his action